pkgs/development/libraries/drogon/default.nix

```diff
@@ -1,4 +1,12 @@
-{ stdenv, fetchFromGitHub, cmake, jsoncpp, libossp_uuid, zlib, openssl, lib }:
+{ stdenv, fetchFromGitHub, cmake, jsoncpp, libossp_uuid, zlib, openssl, lib,
+# miscellaneous
+brotliSupport ? false, brotli,
+c-aresSupport ? false, c-ares,
+# databases
+sqliteSupport ? false, sqlite,
+postgresSupport ? false, postgresql,
+redisSupport ? false, hiredis,
+mysqlSupport ? false, libmysqlclient, mariadb }:
```

Comment:
Which of those can we enable by default? I think brotli, c-are and squlite don't need a flag as they are very small.

Reply:
Yeah I think those are reasonable defaults.